doc/source/whatsnew/v0.19.0.txt

```diff
@@ -427,6 +428,8 @@ Other API changes
 
 Deprecations
 ^^^^^^^^^^^^
+- ``Categorical.reshape`` has been deprecated and will raise an error in a subsequent release (:issue: `12882`)
```

Comment:
no spaces between issue and the number ``(:issue:`12882`)`` it doesn't render correctly.